rtos/TARGET_CORTEX/SysTimer.h

```diff
@@ -119,8 +118,8 @@ class SysTimer: private mbed::TimerEvent, private mbed::NonCopyable<SysTimer> {
     static void _set_irq_pending();
     us_timestamp_t _time_us;
     uint64_t _tick;
-    bool _suspend_time_passed;
-    bool _suspended;
+    bool _suspend_time_passed = true;
```

Comment:
This is C++11, isn't it? We've not activated that yet.

#10104 fixed this in the constructor initializer lists.